docs_guidelines/how_to/add_an_explanation.rst

```diff
@@ -0,0 +1,44 @@
+==================
+Add an explanation
+==================
+
+This guide shows how to add an explanation to a Qiskit repository.
+
+Create the explanation
+======================
+
+In order to create an explanation, you have to go to the  ``docs/explanations`` folder of your Qiskit repository and add a new ``.rst`` file there. In this guide we will assume your explanation is called ``your_explanation.rst``.
+
+Update the page
+===============
+
+Once you have created and written your file, you have to update the ``docs/explanations/index.rst`` file so your new explanation appears in the web page.
+
+
+Update existing section
+-----------------------
+
+If you want to update an existing section, you only need to add this line to the corresponding ``toctree``:
+
+.. code-block:: text
+
+    Title of your explanation <your_explanation>
+
+Make sure your explanation has the same indentation as the other explanations from the ``toctree``.
+
+Create new section
```

Comment:
Yeah at first the idea was saying "if you want to add a guide/tutorial/explanation to an existing section you only have to care about adding one line and not about creating a `toctree`" but it might be better to simply state the full process with the section header and the `toctree` with some existing guides/tutorials/explanations. That way then the user can see the full picture, look at their files and know what's already there and what needs to be added